storage/src/main/java/com/tencent/rss/storage/handler/impl/ComposedClientReadHandler.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Tencent is pleased to support the open source community by making
+ * Firestorm-Spark remote shuffle server available.
+ *
+ * Copyright (C) 2021 THL A29 Limited, a Tencent company.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * https://opensource.org/licenses/Apache-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.tencent.rss.storage.handler.impl;
+
+import com.tencent.rss.common.ShuffleDataResult;
+import com.tencent.rss.storage.handler.api.ClientReadHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ComposedClientReadHandler implements ClientReadHandler {
+
+  private static final Logger LOG = LoggerFactory.getLogger(ComposedClientReadHandler.class);
+
+  private ClientReadHandler hotDataReadHandler;
+  private ClientReadHandler warmDataReadHandler;
+  private ClientReadHandler coldDataReadHandler;
+  private static final int HOT = 1;
+  private static final int WARM = 2;
+  private static final int COLD = 3;
```

Comment:
No frozen handler which read upload handler?

Reply:
Currently, there is MultiStorageReadHandler for both local and hdfs, for memory + MultiStorageReadHandler, only hot & warm will be created.

Reply:
OK, @frankliee If you have time, please implement this function.

Reply:
I will handle it.